src/_includes/components/typestack/typestack.config.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+    title: "Typestack",
+    context: {
+        level: "h1",
+        text: "Typestack"
+    },
```

Comment:
I think the default should be the paragraph. And let's call this component "Typography" rather than "Typestack".